### DISCUSSION TITLE
How to add custom icons to the parallel sprite to use along the default icons?

### DISCUSSION THREAD

**Discussion by delektowski** [eviden-parallel/community#8] [Q&A] [answered]
I need to add custom icons to the parallel sprite to use along the default icons.

Could you tell me how to include my custom icons with Parallel default icons into one sprite?

**Comment by Th3S4mur41** [comment, marked as answer]
The [@atos-parallel/icons package](https://github.com/atos-parallel/icons/pkgs/npm/icons) contains an cli script named `parallel-icons`

The **parallel-icons** script provides a way to generate a custom sprite and also to include additional icons  

> **NOTE**
> See [cli section in the README](https://github.com/atos-parallel/icons#cli) for more details

Let's say you'd want to generate a single sprite containing regular icons and custom icons from your `src/assets` folder. You could do it like this:

```node
parallel-icons -t regular -i src/assets
```

This will generate a **parallel-sprite.svg** file in the public folder (default).
If you'd prefer to create it somewhere else, like let's say the `dist` folder, you can specify that too

```node
parallel-icons -t regular -i src/assets -o dist
```


If you want to include this as a step in your package.json, you could do it as follow:
```package.json
"scripts": {
    "build:sprite": "parallel-icons -t regular -i src/assets -o dist"
}
```
